src/Apps/Collect/__tests__/routes.test.tsx

```diff
@@ -0,0 +1,170 @@
+import { mount } from "enzyme"
+import { Resolver } from "found-relay"
+import createRender from "found/lib/createRender"
+import getFarceResult from "found/lib/server/getFarceResult"
```

Comment:
Would think this unused import would throw an error -- maybe worth commenting out as well, or removing the comments and replacing with a TODO until implementation is fixed.